modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -112,6 +112,9 @@ If you do not add both channels, you cannot create the bootstrap repository, due
 ====
 endif::[]
 
+If you are using modular channels, you must enable the Python 3.6 module stream on the client.
+If you do not provide Python 3.6, the installation of the [package]``spacecmd`` package will fail.
+
```

Comment:
This is true for all RHEL 8 like OSes. The page is named clients-centos.adoc. Is this re-used also fir alma, OL8, Rocky and Native RedHat?

Reply:
> This is true for all RHEL 8 like OSes. The page is named clients-centos.adoc. Is this re-used also fir alma, OL8, Rocky and Native RedHat?

Now we cover: Centos, Oracle Linux, Almalinux, Rocky Linux, SUSE Linux Enterprise Server with Expanded Support, and Red Hat.  SLESES was missing, I just added it.  I hope that that's ok.